server/api/users.js

```diff
@@ -182,6 +213,17 @@ router.post('/users/new', apiLimiter, multer.single('avatar'), async (req, res,
       await dbRef.doc(referrer).collection('referrals').doc(user).create(timestampObj);
     }
 
+    switch (platform) {
+      case 'facebook': {
+        const doc = {
+          [`${platform}UserId`]: platformUserId,
```

Comment:
maybe we can use an object, sth like `${platform}.userId` which (should be) also query-able in firestore
it depends on how we going to get `social/links` though